rest-service/manager_rest/resource_manager.py

```diff
@@ -1408,6 +1444,10 @@ def verify_deployment_parent_labels(self, parents, deployment_id):
         ).items
         _existing_parents = [_parent[0] for _parent in result]
```

Comment:
`_parent[0]` -> `parent.id` ? 
Just for readability.

Reply:
no `parent.id` is available here its just normal list where each item is just string
@YoniItzhak